src/dashboard.ts

```diff
@@ -0,0 +1,180 @@
+import ApexCharts = require('apexcharts');
+
+const pieChart = document.getElementById('pie-chart');
+
+const dataTarget = pieChart.getAttribute('data-target');
+const plantCategory = JSON.parse(dataTarget);
```

Comment:
IMHO, putting JSON inside of html just to get it in JS afterwards feels weird. Why can't we use AJAX fetch to get this data in a separate endpoimt?